File: src/main/java/ru/zdoher/japs/repositories/textbook/TextbookSeriesRepository.java
package ru.zdoher.japs.repositories.textbook;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import ru.zdoher.japs.domain.textbook.TextbookSeries;

@Repository
public interface TextbookSeriesRepository
        extends CrudRepository<TextbookSeries, String> {
}

File: src/main/java/ru/zdoher/japs/repositories/textbook/TextbookSeriesRepository.kt
package ru.zdoher.japs.repositories.textbook

import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import ru.zdoher.japs.domain.textbook.TextbookSeries

@Repository
interface TextbookSeriesRepository : ReactiveMongoRepository<TextbookSeries, String>